src/main/java/com/adobe/aem/compgenerator/utils/DialogUtils.java

```diff
@@ -264,7 +265,19 @@ private static void processItems(Document document, Node itemsNode, Property pro
             }
 
             if (StringUtils.equalsIgnoreCase("multifield", property.getType())) {
-                optionNode.setAttribute(Constants.PROPERTY_NAME, "./" + item.getField());
+                getPrimaryFieldName(item).ifPresent(name -> {
+                    optionNode.setAttribute(Constants.PROPERTY_NAME, name);
```

Comment:
Is cq_msm_lockable not applicable to fields within a multifield?

Reply:
I'm not sure. I didn't want to make an opinionated change on that without knowing either way.

Reply:
Ah I suppose it's at least no different than how it had been working.